README.md

```diff
@@ -117,7 +117,7 @@ $ rails new . -d postgresql
 ```
 
 ```
-Your answer...
+"$ rails new tunr" creates a rails directory called tunr and connects the tunr application to mysql. On the other hand, "$ rails new . -d postgresql" creates a rails file within a directory so that the app can run in psql. That is, it connects to the psql database.
```

Comment:
:+1:

The default database is Sqlite.
Technically, "psql" is the command line app we use to query a Postgresql database.